samples/golang/mywebapi/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.10
+
+RUN mkdir -p /go/src
+
+COPY src /go/src/
+WORKDIR /go/src/app
+
+RUN go install app
+EXPOSE 80
```

Comment:
Put the `EXPOSE 80` statement right after the FROM, because it is likely to never change. As it is currently, whenever source code changes, it will need to run this extra step again. You want to minimize the number of Docker instructions that have to run after copying in all the source code.